Resources/config/security.xml

```diff
@@ -9,8 +9,7 @@
         </service>
         <service id="sonata.media.security.superadmin_strategy" class="Sonata\MediaBundle\Security\RolesDownloadStrategy">
             <argument type="service" id="translator"/>
-            <argument/>
-            <!-- Either security.context or security.authorization_checker -->
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
This looks wrong. I think this empty argument is supposed to be replaced in `SonataMediaExtension`